Sprint 13

kazakovdmitriy opened the pull request.

Commit:
Выполнен рефакторинг WebViewViewController, созданы отдельные сущности WebViewPresenter, AuthHelper, AuthConfiguration

Commit:
Добавил тесты для WebView

Commit:
Выполнен рефактор ProfileViewController и ImageListViewController

Commit:
Написаны тесты для imagelistviewcontroller

DanilaG commented on ImageFeed/Resources/AuthConfiguration.swift:
```diff
@@ -0,0 +1,46 @@
+        self.authURLString = authURLString
+    }
+    
+    static var standart: AuthConfiguration {
```
Сейчас, к одним и тем же данным можно получить доступ через `Constants` и через `AuthConfiguration.standart`. Что при дальнейшей разработке/поддержки программы пожет приводить к путанице - что выбрать `Constants` или `AuthConfiguration.standart`. Для избавления от путаницы, рекомендую убрать дублирующиеся константы из  `Constants`

Commit:
Fix
Сделал ссылку на view слабой в ProfileViewPresenter
Исправил перенос строк в bio профиля
Сделал lastName в ProfileResult опциональным

Commit:
Разделил AuthConfigurator и Constants
Поправил отображение лейбла fullName
Добавил в testProfile открытие экрана авторизации

Commit:
Fix 2
Сделал форматирование unit тестов

kazakovdmitriy commented:
Внес исправления по замечаниям, отправил новую ссылку на проверку

Commit:
Удалил лишние сториборды
